### DISCUSSION TITLE
Woocommerce subscription upgrade prorata not working

### DISCUSSION THREAD

**Discussion by kickmouth** [bootscore/bootscore#680] [Q&A]
Hi @crftwrk 
Using bootscore and woocommerce subscriptions to sign up members to our theatre: https://www.towertheatre.org.uk/
There are two tiers of paid for membership - the cheaper "friend" option and a premium "member" option.
Using the woocommerce subscription plugin, if a Friend wants to upgrade to a full Member they should be charged the prorata difference in price. As it did when we used the Storefront theme. Since switching to bootscore this is no longer possible and woo support say that it is due specifically to a conflict with Bootscore.
Has anyone else experienced this and is there a workaround? Should I disable all Ajax maybe?
Thanks

**Comment by crftwrk** [comment]
Hi kickmouth,

we had this already here https://github.com/bootscore/bootscore/issues/564.

The Ajax cart is not able to switch already paid products to a different membership/price. It's possible to switch during the process, but not if they already paid https://dev.bootscore.me/product/subscription-variable/.

In your case, you have to disable Ajax cart as described here https://bootscore.me/documentation/woocommerce/#Disable_AJAX_cart. Now theme has the same functionality like the Storefront theme and subscriptions should work as expected.

**Reply by kickmouth** [reply]
Apologies @crftwrk  - my search for the issue wasn't thorough enough - but thanks for the swift reply.
Testing it out now. I'll let you know
:)
